tests/background.py

```diff
@@ -361,7 +361,76 @@ def testCFHT_oldAPI(self):
         if display:
             ds9.mtv(mi, frame = 1)
 
-            
+    def testXY0(self):
+        """Test fitting the background to an image with nonzero xy0
+
+        The statsImage and background image should not vary with xy0
+        """
+        bgImageList = [] # list of background images, one per xy0
+        statsImageList = [] # list of stats images, one per xy0
+        for xy0 in (afwGeom.Point2I(0, 0), afwGeom.Point2I(1000, 500)):
+            afwdataDir = eups.productDir("afwdata")
+            if not afwdataDir:
+                print >> sys.stderr, "Skipping testCFHT as afwdata is not setup"
+                return
```

Comment:
Shouldn't we use `unittest.skipIf` or similar here?

I notice that this pattern is reproduced through this file, both in the other test you've introduced (`testSubImage`) and elsewhere. It would be far nicer to just run the check for `afwdata` once and then skip the tests that require it, rather than repeating it in every test.

Also, this is not `testCFHT`.